### ISSUE TITLE
[BUG] Constant migration notification

### ISSUE THREAD

**Issue by apoapostolov** [Larkinabout/fvtt-token-action-hud-core#147]
Each time the world is reloaded, a notification informs of migrating tags. This might be a bug (flags are migrated once, but notification keeps showing) or maybe flags aren't properly set. FVTT 11 + PF2 5.0.0 beta2

![image](https://github.com/Larkinabout/fvtt-token-action-hud-core/assets/4083812/1bd95113-05ba-489e-8ef0-0ceb62002853)

**Comment by Larkinabout** [comment]
In your case, run the following as a macro script:

```js
const actors = game.actors.filter(actor => actor.getFlag('token-action-hud-core', 'categories'))
for (const actor of actors) {
    actor.unsetFlag('token-action-hud-core', 'categories')
}

const users = game.users.filter(user => user.getFlag('token-action-hud-core', 'categories'))
for (const user of users) {
    user.unsetFlag('token-action-hud-core', 'categories')
    user.unsetFlag('token-action-hud-core', 'default')
}
```

It'll clear out the old flags on all users and actors.

**Comment by Suldrun45** [comment]
Same issue on a few worlds (pf2e, latest version), I ran the script above but it doesn't change anything

**Comment by apoapostolov** [comment]
Same the macro did not work.

**Comment by Suldrun45** [comment]
It just says "undefined" when I do that

**Comment by Larkinabout** [comment]
The script has cleared out the old flags then or the original migration did. What's confusing me is that the first thing the migration does it check whether those flags exist and if they don't it exits the migration. I don't seem to be able to reproduce on my end at all, but I might consider dropping the migration entirely in the next version anyway.

**Comment by Larkinabout** [comment]
@ogsfernandes, do you get an red errors in the browser console? A failure to restore the HUD layout on reload suggests it's either failing to save or retrieve the data from the json files.

**Comment by Larkinabout** [comment]
Fixed in Token Action HUD Core 1.4.10.
